Replace cluster with addCluster

theotimeee opened the pull request:
After mia PR https://github.com/microbiome/mia/pull/502 renaming `cluster` to `addCluster` in the mia package, this PR replaces `cluster` with `addCluster` in the miaViz package.

Commit:
Replace cluster with addCluster after renaming in mia package

Commit:
Merge devel

antagomir commented on R/plotDMN.R:
```diff
@@ -54,8 +54,8 @@ setMethod("plotDMNFit", signature = c(x = "SummarizedExperiment"),
-            .Deprecated(old="getDMN", new="cluster", 
-                    "Now runDMN and calculateDMN are deprecated. Use cluster with DMMParam parameter and full parameter set as true instead.")
+            .Deprecated(old="getDMN", new="addCluster", 
+                    "Now runDMN and calculateDMN are deprecated. Use addCluster with DMMParam parameter and full parameter set as true instead.")
```
could be splitted on two rows.

antagomir reviewed (approved):
Seems good to me overall.

antagomir commented:
Conflicts to resolve.

Commit:
merge devel

Merge branch 'devel' of https://github.com/microbiome/miaViz into rename_cluster

# Conflicts:
#	NEWS

TuomasBorman committed:
Update plotDMN.R

TuomasBorman committed:
up

antagomir commented:
up

Commit:
add magick package to dependencies to solve vignette building fail

Merge branch 'devel' of https://github.com/microbiome/miaViz into rename_cluster

# Conflicts:
#	NEWS

Commit:
fix ponctuation

Commit:
remove magick from dependencies

TuomasBorman committed:
Merge branch 'devel' into rename_cluster

TuomasBorman committed:
Update NEWS

TuomasBorman committed:
Update plotDMN.R

TuomasBorman committed:
Update plotDMN.R

TuomasBorman committed:
Merge branch 'devel' into rename_cluster

TuomasBorman committed:
Merge branch 'devel' into rename_cluster

TuomasBorman committed:
up
